modules/ml/test/test_bayes.cpp

```diff
@@ -0,0 +1,53 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "test_precomp.hpp"
+
+namespace opencv_test { namespace {
+
+TEST(ML_NBAYES, regression_5911)
+{
+    int N=12;
+    Ptr<ml::NormalBayesClassifier> nb = cv::ml::NormalBayesClassifier::create();
+
+    // data:
+    Mat_<float> X(N,4);
+    X << 1,2,3,4,  1,2,3,4,   1,2,3,4,    1,2,3,4,
+         5,5,5,5,  5,5,5,5,   5,5,5,5,    5,5,5,5,
+         4,3,2,1,  4,3,2,1,   4,3,2,1,    4,3,2,1;
```

Comment:
Lets avoid bloating of binary code and slowdown compiler to generate that.

```
    float X_data[] = {
        1,2,3,4,  1,2,3,4,   1,2,3,4,    1,2,3,4,
        5,5,5,5,  5,5,5,5,   5,5,5,5,    5,5,5,5,
        4,3,2,1,  4,3,2,1,   4,3,2,1,    4,3,2,1
    };
    Mat_<float> X(N,4, X_data);
```

(58Kb -> 50Kb for .o file: 1.7Kb of filling code + exception tables and other code helpers)